src/test/libraryUtilitiesTestClasses.ts

```diff
@@ -0,0 +1,15 @@
+export class typeListNodeData {
+    fullyQualifiedName: any;
+    iconName: any;
+    creationName: any;
+    itemType: any;
+}
+
+
+export class layoutElementData {
+    text: any;
+    iconName: any;
+    elementType: any;
+    include: any;
+    childElements: layoutElementData[] = [];
+}
```

Comment:
We may want Pascal case for classes, and as much as possible, use type information. E.g. many of those `any` should really be `string`.